ui/app/src/components/SplitButton/SplitButton.tsx

```diff
@@ -81,7 +82,9 @@ export function SplitButton(props: SplitButtonProps) {
   return (
     <>
       {loading ? (
-        <PrimaryButton loading disabled />
+        <PrimaryButton loading disabled>
```

Comment:
Fix this at the SplitButtonUI level to use the MUI `loading` prop of button and freeze the buttons at that level.